tests/lean/semicolonOrLinebreak.lean.expected.out

```diff
@@ -0,0 +1,5 @@
+building '/nix/store/kzswn86bwsdhxg5jb5d8vnfjkq36akl7-lean.drv'...
+semicolonOrLinebreak.lean:2:12: error: expected ';' or line break
+semicolonOrLinebreak.lean:9:31-10:8: error: field 'y' is missing
+semicolonOrLinebreak.lean:10:8: error: expected command
+semicolonOrLinebreak.lean:20:8: error: expected ':'
```

Comment:
This message is "fun": the longest parse `{ x y` is not from the structure instance notation, but from the dep arrow `{ x y : _ } -> _`, thus the `:` reference. Not much we can do there, but I don't think it matters in practice.